apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1086,11 +1086,14 @@ export class CaseService {
       })
     }
 
+    // TODO: Use subpoenas already included in theCase.defendants
+    // - no need to call the subpoena service
+    // - then add to transition tests
```

Comment:
All subpoenas are already accessible from the case itself so there is no need to make an additional database call.